indico/modules/categories/controllers/management.py

```diff
@@ -64,8 +67,22 @@ def _category_query_options(self):
         return children_strategy,
 
     def _process(self):
+        page = request.args.get('page', '1')
+        order_columns = {'start_dt': Event.start_dt, 'title': db.func.lower(Event.title)}
+        direction = 'desc' if request.args.get('desc', '1') == '1' else 'asc'
+        order_column = order_columns[request.args.get('order', 'start_dt')]
+        query = (Event.query.with_parent(self.category)
+                 .order_by(getattr(order_column, direction)())
+                 .order_by(Event.id))
+        if page == 'all':
+            query = query.paginate(show_all=True)
```

Comment:
the paginated object is not a query anymore. i'd call this `events` or `event_pager` or something like that